examples/pinn_pde_solver/CMakeLists.txt

```diff
@@ -7,5 +7,5 @@ target_include_directories(phasm-example-pinn-pdesolver PRIVATE
 #        ${CMAKE_CURRENT_SOURCE_DIR}/../../surrogate/include
 #        ${CMAKE_CURRENT_SOURCE_DIR}/../../memtrace/include
         )
-target_link_libraries(phasm-example-pinn-pdesolver ${TORCH_LIBRARIES} phasm-surrogate)
```

Comment:
What failed when you tried to link this? I'm attempting to put together a new Docker container where all PHASM components build together (minus CUDA), and I'm going to definitely need your help for the CUDA version.

Reply:
Nothing failed I guess... Just want to clear some code.